joomla/joomla-cms
# TinyMCE tolbar builder: use TinyMCE translation for the TinyMCE buttons

Pull Request for Issue #13343

### Summary of Changes
This pull make the builder to use TinyMCE translation for the TinyMCE buttons

### Testing Instructions
Set the admin language to different from English, and check that 99% of button are translated already, in the Plugin -TinyMCE.

### Documentation Changes Required
nope
